tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/views/execution_data/execution_data_container.ts

```diff
@@ -0,0 +1,106 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+import {Component} from '@angular/core';
+import {select, Store, createSelector} from '@ngrx/store';
+
+import {Execution, State, TensorDebugMode} from '../../store/debugger_types';
+
+import {
+  getActiveRunId,
+  getFocusedExecutionData,
+  getFocusedExecutionIndex,
+} from '../../store';
+
+/** @typehack */ import * as _typeHackRxjs from 'rxjs';
+
+@Component({
+  selector: 'tf-debugger-v2-execution-data',
+  template: `
+    <execution-data-component
+      [activeRunId]="activeRunId$ | async"
+      [focusedExecutionIndex]="focusedExecutionIndex$ | async"
+      [focusedExecutionData]="focusedExecutionData$ | async"
+      [tensorDebugModeName]="tensorDebugModeName$ | async"
+      [anyDebugTensorValues]="anyDebugTensorValues$ | async"
+      [debugTensorValues]="debugTensorValues$ | async"
+    ></execution-data-component>
+  `,
+})
+export class ExecutionDataContainer {
+  readonly activeRunId$ = this.store.pipe(select(getActiveRunId));
+
+  readonly focusedExecutionIndex$ = this.store.pipe(
+    select(getFocusedExecutionIndex)
+  );
+
+  readonly focusedExecutionData$ = this.store.pipe(
+    select(getFocusedExecutionData)
+  );
+
+  readonly tensorDebugModeName$ = this.store.pipe(
```

Comment:
Wouldn't it be better to just move enum values around instead of enum name? If you are going to ever show enum as string in the UI, I would rather want template to have enum to displayName conversion (for i18n, too).

Reply:
Good suggestion. I do want the "debug mode" field in the UI to show the text name, which is much easier to understand than the numerical enum value. I made the change so that this property returns a numerical value and the to-string conversion is done in the html template.